### PR TITLE
chore(qa): Prepare manifest for TheAnvil staging environments (staging)

### PR DESCRIPTION
**Pull request by themarcelor**
Preparing 1st release for TheAnvil.

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# staging.theanvil.io/manifest.json
## Deployment changes
- hatchery
  - New `pull_policy` option is available for `hatchery.containers` entries, eg: (uc-cdis/hatchery#1)
  - ``` (uc-cdis/hatchery#1)
  - { (uc-cdis/hatchery#1)
  - "target-port": 8787, (uc-cdis/hatchery#1)
  - "cpu-limit": "1.0", (uc-cdis/hatchery#1)
  - "memory-limit": "512Mi", (uc-cdis/hatchery#1)
  - "pull_policy": "Always", (uc-cdis/hatchery#1)
  - "name": "R Studio", (uc-cdis/hatchery#1)
  - "image": "heliumdatastage/rstudio-server:1", (uc-cdis/hatchery#1)
  - "env": { (uc-cdis/hatchery#1)
  - "DISABLE_AUTH": "true" (uc-cdis/hatchery#1)
  - }, (uc-cdis/hatchery#1)
  - "args": [], (uc-cdis/hatchery#1)
  - "path-rewrite": "/", (uc-cdis/hatchery#1)
  - "use-tls": "false", (uc-cdis/hatchery#1)
  - "ready-probe": "/" (uc-cdis/hatchery#1)
  - } (uc-cdis/hatchery#1)
  - ``` (uc-cdis/hatchery#1)
- pidgin
  - added required configuration variable "API_HEALTH_URL" for status endpoint (uc-cdis/pidgin#31)
  - use the latest version of the python 3 base image (uc-cdis/pidgin#29)
  - The Dockerfile now uses an Alpine image (uc-cdis/pidgin#24)
- sheepdog
  - Fix Travis coverage reports (combine the results of all 3 tests into one coverage report). (uc-cdis/sheepdog#314)
  - ARBORIST_URL environment variable replaces broken default ARBORIST config variable (uc-cdis/sheepdog#304)
  - Optional configuration variable `ARBORIST` (arborist base URL) (uc-cdis/sheepdog#298)
  - Requires Arborist to be deployed (uc-cdis/sheepdog#298)
  - look for INDEX_CLIENT instead of SIGNPOST in app.config (uc-cdis/sheepdog#291)
  - get INDEX_CLIENT_HOST env var instead of SIGNPOST_HOST (uc-cdis/sheepdog#291)
- spark
  -  (uc-cdis/spark#10)

**Comment by PlanXCyborg** [comment]
# staging.theanvil.io/manifest.json
## Deployment changes
- hatchery
  - New `pull_policy` option is available for `hatchery.containers` entries, eg: (uc-cdis/hatchery#1)
  - ``` (uc-cdis/hatchery#1)
  - { (uc-cdis/hatchery#1)
  - "target-port": 8787, (uc-cdis/hatchery#1)
  - "cpu-limit": "1.0", (uc-cdis/hatchery#1)
  - "memory-limit": "512Mi", (uc-cdis/hatchery#1)
  - "pull_policy": "Always", (uc-cdis/hatchery#1)
  - "name": "R Studio", (uc-cdis/hatchery#1)
  - "image": "heliumdatastage/rstudio-server:1", (uc-cdis/hatchery#1)
  - "env": { (uc-cdis/hatchery#1)
  - "DISABLE_AUTH": "true" (uc-cdis/hatchery#1)
  - }, (uc-cdis/hatchery#1)
  - "args": [], (uc-cdis/hatchery#1)
  - "path-rewrite": "/", (uc-cdis/hatchery#1)
  - "use-tls": "false", (uc-cdis/hatchery#1)
  - "ready-probe": "/" (uc-cdis/hatchery#1)
  - } (uc-cdis/hatchery#1)
  - ``` (uc-cdis/hatchery#1)
- pidgin
  - added required configuration variable "API_HEALTH_URL" for status endpoint (uc-cdis/pidgin#31)
  - use the latest version of the python 3 base image (uc-cdis/pidgin#29)
  - The Dockerfile now uses an Alpine image (uc-cdis/pidgin#24)
- sheepdog
  - Fix Travis coverage reports (combine the results of all 3 tests into one coverage report). (uc-cdis/sheepdog#314)
  - ARBORIST_URL environment variable replaces broken default ARBORIST config variable (uc-cdis/sheepdog#304)
  - Optional configuration variable `ARBORIST` (arborist base URL) (uc-cdis/sheepdog#298)
  - Requires Arborist to be deployed (uc-cdis/sheepdog#298)
  - look for INDEX_CLIENT instead of SIGNPOST in app.config (uc-cdis/sheepdog#291)
  - get INDEX_CLIENT_HOST env var instead of SIGNPOST_HOST (uc-cdis/sheepdog#291)
- spark
  -  (uc-cdis/spark#10)

File: staging.theanvil.io/manifest.json [hunk 1]
```diff
@@ -7,27 +7,25 @@
...
-    "spark": "quay.io/cdis/gen3-spark:1.0.0",
-    "tube": "quay.io/cdis/tube:0.3.18",
-    "ssjdispatcher": "quay.io/cdis/ssjdispatcher:0.0.6",
-    "wts": "quay.io/cdis/workspace-token-service:0.2.0"
+    "sheepdog": "quay.io/cdis/sheepdog:2020.02",
+    "sower": "quay.io/cdis/sower:2020.02",
+    "spark": "quay.io/cdis/gen3-spark:2020.02",
+    "tube": "quay.io/cdis/tube:2020.02",
+    "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2020.02",
+    "wts": "quay.io/cdis/workspace-token-service:2020.02"
```
Comment (m0nhawk) [inline staging.theanvil.io/manifest.json]: Why not `2020.03`?

File: staging.theanvil.io/manifest.json [hunk 2]
```diff
@@ -110,6 +173,138 @@
         "path-rewrite": "/",
         "use-tls": "false",
         "ready-probe": "/"
+      },
+      {
+        "target-port": 8888,
+        "cpu-limit": "1.0",
+        "memory-limit": "512Mi",
+        "name": "Jupyter Notebook Bio Python",
+        "image": "quay.io/occ_data/jupyternotebook:1.9.0",
```
Comment (m0nhawk) [inline staging.theanvil.io/manifest.json]: We should move to `uc-cdis` images from [`containers`](https://github.com/uc-cdis/containers).

File: staging.theanvil.io/manifest.json [hunk 3]
```diff
@@ -110,6 +173,138 @@
...
+        "user-uid": 1000,
+        "fs-gid": 100,
+        "user-volume-location": "/home/jovyan/pd"
+      },
+      {
+        "target-port": 8888,
+        "cpu-limit": "4.0",
+        "memory-limit": "15512Mi",
+        "name": "Jupyter Notebook Power Python",
+        "image": "quay.io/occ_data/jupyternotebook:1.7.2",
```
Comment (m0nhawk) [inline staging.theanvil.io/manifest.json]: We should move to `uc-cdis` images from [`containers`](https://github.com/uc-cdis/containers) (maybe not now, but eventually).
Why the old version of image?

File: staging.theanvil.io/manifest.json [hunk 4]
```diff
@@ -110,6 +173,138 @@
...
+        "user-uid": 1000,
+        "fs-gid": 100,
+        "user-volume-location": "/home/jovyan/pd"
+      },
+      {
+        "target-port": 8888,
+        "cpu-limit": "1.0",
+        "memory-limit": "8096Mi",
+        "name": "Helium Autoencoder Demo",
+        "image": "quay.io/cdis/auntoencoder-copd-demo:latest",
```
Comment (m0nhawk) [inline staging.theanvil.io/manifest.json]: Images should be pinned no matter what.

File: staging.theanvil.io/manifest.json (hunk 1 above)
Comment (themarcelor) [reply staging.theanvil.io/manifest.json]: That's coming up next.

**Pull request merged**